release/build-common.xml

```diff
@@ -9,7 +9,6 @@
     <!-- A Java 9 or later compiler is required -->
 
     <exec executable="java_home" outputproperty="jdk">
-        <arg value="-F"/>
```

Comment:
Including the -F never builds for me, because java_home -F doesn't obey the plus in 9+ and fails with any jdk other than version 9.*.